nixos/modules/services/monitoring/grafana.nix

```diff
@@ -622,6 +629,8 @@ in {
       serviceConfig = {
         WorkingDirectory = cfg.dataDir;
         User = "grafana";
+        RuntimeDirectory = "grafana";
```

Comment:
What is stored in grafanas runtime directory, or what's the reason for adding this?

Reply:
I set the default path of the unix socket in line 297 to `/run/grafana/grafana.sock` (only used if unix sockets are enabled). Of course the default could also be something like `/run/grafana.sock` when the runtime directory shouldn't be created.

Reply:
Sorry I was confusint this with the `PrivateTmp` option, which wouldn't have made sense if the socket was stored there. `RuntimeDirectory` of course makes sense to make the socket available.